tests/csrf_tests/tests.py

```diff
@@ -352,6 +352,20 @@ def test_https_good_referer_2(self):
         req2 = CsrfViewMiddleware().process_view(req, post_form_view, (), {})
         self.assertIsNone(req2)
 
+    @override_settings(ALLOWED_HOSTS=['www.example.com'], CSRF_TRUSTED_ORIGINS=['dashboard.example.com'])
+    def test_https_csrf_trusted_origin_allowed(self):
+        """
+        Test that a POST HTTPS request with a referer added to the
+        CSRF_TRUSTED_ORIGINS setting is accepted
+        """
+        # See ticket #24496
```

Comment:
I don't know that the ticket reference is necessary. We try to reserve it for obscure issues that can't be easily captured in a docstring.